docs/05-technical-reference/00-custom-resources/ac-01-application.md

```diff
@@ -45,25 +45,23 @@ This table lists all the possible parameters of a given resource together with t
-| **spec.services.entries** | Yes | Contains the information about the APIs and events that the service offered by the Application provides. Parameter provided by Application Registry, do not edit. |
-| **spec.services.entries.type** | Yes | Specifies the entry type: API or event. Parameter provided by Application Registry, do not edit. |
-| **spec.services.entries.centralGatewayUrl** | No | Specifies the URL of Application Gateway. Internal address resolvable only within the cluster. This field is required for the API entry type. Parameter provided by Application Registry, do not edit. |
+| **spec.services.name** | No | Represents a unique name of the service. Used for proxying in Application Gateway. |
+| **spec.services.providerDisplayName** | Yes | Specifies a human-readable name of the Application service provider. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |
+| **spec.services.tags** | No | Specifies additional tags used for better documentation of the available APIs. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |
+| **spec.services.labels** | No | Specifies additional labels for the service offered by the Application. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |
+| **spec.services.entries** | Yes | Contains the information about the APIs and events that the service offered by the Application provides. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |
+| **spec.services.entries.type** | Yes | Specifies the entry type: API or event. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |
+| **spec.services.entries.centralGatewayUrl** | No | Specifies the URL of Application Gateway. Internal address resolvable only within the cluster. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |
```

Comment:
```suggestion
| **spec.services.entries.centralGatewayUrl** | No | Specifies the URL of Application Gateway. The internal address is resolvable only within the cluster. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

Reply:
Also: is `resolvable` a good word here? 🤔

Reply:
Yep. And address that can be resolved only within the cluster = is resolvable only within the cluster. 
I'll apply the `mode` bit, but I'll keep the original description, because it's the second part of the description of the parameter, not additional info.